wiki/Beatmaps/en.md

```diff
@@ -3,57 +3,153 @@
+**Beatmaps** are game levels created for songs and stored in text files with [the `.osu` extension](/wiki/osu!_File_Formats/Osu_(file_format)). They have different structure and sometimes may be played only in one of osu!'s [game modes](/wiki/Game_mode).
 
-Beatmaps are made as part of **[beatmapsets](Beatmapsets)**. Beatmapsets group beatmaps together so that they can be [submitted](/wiki/Glossary#bss) to the website and show up on the [beatmap listing](https://osu.ppy.sh/beatmapsets). They typically cover one song, where each beatmap of the set is a different [difficulty](/wiki/Difficulties).
+## Overview
+
+At the core, a beatmap consists of [hit objects](/wiki/Hit_object) of a certain [game mode](/wiki/Game_mode), which are set up according to [timing](/wiki/Glossary#timing-section) tailored to a specific song. Beatmaps have different aspects, which directly affect user experience.
+
+The hit objects of a beatmap compose [patterns](Pattern), which also vary per game mode and set the tone for overall difficulty, and how a beatmap should be played. The process of playing the beatmap is altered by various [difficulty settings](/wiki/Beatmap_Editor/Song_Setup#difficulty) used by the beatmap's author and [game modifiers](/wiki/Game_modifier) toggled by a player.
+
+From the auditory perspective, a beatmap can augment the song by providing one or multiple sets of audio samples, or [hitsounds](/wiki/Beatmapping/Hitsound), that serve as an additional layer of feedback.
+
+Visual components of a beatmap, contrary to its patterns, provide auxiliary experience during gameplay and can be disabled in the [visual settings](/wiki/Visual_Settings) popup for the most part. They include:
+
+- [Background imagery](Background) and video, which both can act as a playfield;
+- [Kiai time](/wiki/Kiai_time) and [combo colours](/wiki/Glossary/Combo_colour) that help distinguish different beatmap sections from each other;
+- [Storyboard](/wiki/Storyboards) that adds motion graphics and special effects, or provides a background story or theme for the song;
+- [Custom skin](/wiki/Skinning), which changes the appearance of interface and gameplay elements;
+- [Title text](Title_Text), which presents the song's metadata.
+
+## Beatmapsets
+
+*Main article: [Beatmapsets](Beatmapsets)*
+
+On a higher level, beatmaps are grouped into [beatmapsets](Beatmapsets) and then [submitted](/wiki/Glossary#bss) to the [beatmap listing](https://osu.ppy.sh/beatmapsets). Each beatmapset typically covers one song and holds together several beatmaps with different object placement and gameplay settings. In this context, a beatmap is called a [difficulty](/wiki/Difficulties).
+
+While every beatmapset is attributed to [a single person](Beatmapsets/Beatmapset_host), it is often a joint effort: some difficulties may be done by other people working [in collaboration](Beatmap_collaborations) or [separately](Beatmapsets/Guest_difficulty).
```

Comment:
i think this line can be removed in order not to repeat the same information as in the `Beatmapsets` article

Reply:
I consider it important enough to be mentioned here as well, at least briefly